### PR TITLE
docs: Add 21YunBox as Hosting Provider Guides

### PR DESCRIPTION
**Pull request by tobyglei**
### What does it do?

Add Hosting Provider Guides for 21YunBox

### Why is it needed?

The current hosting providers on the documents don't work in China because the Great Firewall of China. 21YunBox is here to solve the problem.

### Related issue(s)/PR(s)

na.

### CODE REVIEW

**Comment by tobyglei** [comment]
can anyone review this? Thanks

**Comment by derrickmehaffy** [comment]
@tobyglei the SVG you are using appears to be clipped: 

image

File: docs/v3.x/deployment/21yunbox.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+## Setup
+
+You can set up a Strapi CMS on 21YunBox in four quick steps:
+
+1. Create a 21YunBox Account. Visit [21YunBox dashboard](https://https://www.21yunbox.com/u/signup/) to create an account if you don't already have one.
+2. Create a new web service on 21YunBox, and give 21YunBox permission to access your GitHub or Gitee repo.
+3. Use the following values during creation:
+
+   |                       |                                                  |
```
Comment (derrickmehaffy) [inline docs/v3.x/deployment/21yunbox.md:37]: Please add headers to your table

File: docs/v3.x/deployment/21yunbox.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+   | --------------------- | ------------------------------------------------ |
+   | **Environment**       | `Node 12.19`                                    |
+   | **Build Command**     | `yarn && yarn build` (or your own build command) |
+   | **Publish Directory** | `rsync -a public/ /data/public/ && yarn start` (or your own output directory)        |
+
+4. Add the following environment variables, then press the "Deploy" button.
+
+   |                       |                                                  |
```
Comment (derrickmehaffy) [inline docs/v3.x/deployment/21yunbox.md:45]: Also, needs table headers

File: docs/v3.x/getting-started/deployment.md [hunk 3]
```diff
@@ -10,6 +10,18 @@ Deploying **databases** along with Strapi is covered in the [Databases Guide](..
...
... values="0 0 0 0 1.000000 0 0 0 0 1.000000 0 0 0 0 1.000000 0 0 0 1.000000 0"/></filter></defs><g fill="none" fill-rule="evenodd"><g fill-rule="nonzero"><path fill="#FA598D" d="M41.325 13.897h5.303v36.206h-5.303zM10.331 13.897h5.303V19.2h-5.303zM17.371 13.897h5.303V19.2h-5.303zM24.502 13.897h5.303V26.24h-5.303z"/><path fill="#00E5A2" d="M10.331 28.891h19.474v5.303H10.331z"/><path fill="#FA598D" d="M10.331 36.846h5.303v13.257h-5.303zM17.371 44.8h5.303v5.303h-5.303zM24.502 44.8h5.303v5.303h-5.303z"/><path fill="#FDC0A5" d="M34.194 13.897h5.303V19.2h-5.303z"/><path fill="#FA598D" d="M41.325 44.8h5.303v5.303h-5.303z"/><path fill="#31198C" d="M48.365 44.8h5.303v5.303h-5.303zM34.194 44.8h5.303v5.303h-5.303z"/></g><g filter="url(#a)" transform="translate(10.331 13.897)"><g fill="#FFF" fill-rule="nonzero"><path d="M30.994 0h5.303v36.206h-5.303zM0 0h5.303v5.303H0zM7.04 0h5.303v5.303H7.04zM14.171 0h5.303v12.343h-5.303zM0 14.994h19.474v5.303H0zM0 22.949h5.303v13.257H0zM7.04 30.903h5.303v5.303H7.04zM14.171 30.903h5.303v5.303h-5.303zM23.863 0h5.303v5.303h-5.303z"/><path d="M30.994 30.903h5.303v5.303h-5.303zM38.034 30.903h5.303v5.303h-5.303zM23.863 30.903h5.303v5.303h-5.303z"/></g></g></g></svg>
```
Comment (derrickmehaffy) [inline docs/v3.x/getting-started/deployment.md:16]: As mentioned in my comment, your SVG is being clipped, you may need to adjust the SVG to get it to fit properly.

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]

**Comment by derrickmehaffy** [comment]
We are in the process of moving to a new Documentation repo, we may need to close this PR and have it reopened on the new repo here: https://github.com/strapi/documentation unless the changes can be made in the next 24 hours.

**Comment by derrickmehaffy** [comment]
Closing, please take my review comments and submit a PR to the new documentation repo.

**Pull request closed without merging**